js/client-chat.js

```diff
@@ -1670,7 +1692,13 @@
 			var speakerHasAuth = " +\u2606".indexOf(name.charAt(0)) < 0;
 			var user = (this.users && this.users[app.user.get('userid')]) || {};
 			var readerHasAuth = !" +\u2606\u203D!".includes(user.group || ' ');
-			if (app.ignore[userid] && !speakerHasAuth && !readerHasAuth) return;
+			if (app.ignore[userid] && !speakerHasAuth && !readerHasAuth) {
+				if (!app.ignore[userid].notified) {
+					this.$chat.append('<div class="chat">1 message from user ' + name + ' ignored. (to unignore use /unignore)</div>');
```

Comment:
The number of messages is kind of irrelevant if you stop notifying after the first.

```suggestion
					this.$chat.append('<div class="chat">A message from ' + BattleLog.escapeHTML(name) + ' was ignored. (to unignore use /unignore)</div>');
```